docs/man/man1/xhc-hb04.1

```diff
@@ -108,91 +100,83 @@ Example:
+require: \-9999.999 <= value <= 9999.999.
 .TP
 (float in) \fIxhc\-hb04.[xyza].pos\-relative\fR Relative position display.
-(typically connect to: halui.axis.N.pos\-relative). The LCD display
-for pos\-relative is fixed format with a sign, 4 number digits and 3
-fraction digits (+XXXX.XXX), require: \-9999.999 <= value <= 9999.999.
+(typically connect to: halui.axis.N.pos\-relative).
+The LCD display for pos\-relative is fixed format with a sign, 4 number digits and 3 fraction digits (+XXXX.XXX),
+require: \-9999.999 <= value <= 9999.999.
 
 .TP
 (float in) \fIxhc\-hb04.feed\-override\fR Feed\-override value.
-The float value is converted to a 16 bit integer and multiplied by 100 in
-order to display as percent, require: 0 <= pinvalue <= 655
+The float value is converted to a 16 bit integer and multiplied by 100 in order to display as percent, require: 0 <= pinvalue <= 655
 (typically connect to: halui.feed\-override.value)
 .TP
 (float in) \fIxhc\-hb04.feed\-value\fR Current Feed-value (units/sec).
-The float value is converted to a 16 bit integer and multiplied by 60 in
-order to display as units-per-minute, require: 0 <= pinvalue <= 1092
+The float value is converted to a 16 bit integer and multiplied by 60 in order to display as units-per-minute,
+require: 0 <= pinvalue <= 1092
 (65520 units-per-minute) (typically connect to: motion.current\-vel)
 
 .TP
 (float in) \fIxhc\-hb04.spindle\-override\fR Spindle\-override value.
-The float value is converted to a 16 bit integer and multiplied by 100 in
-order to display as percent, require: 0 <= pinvalue <= 655)
+The float value is converted to a 16 bit integer and multiplied by 100 in order to display as percent, require: 0 <= pinvalue <= 655)
 (typically connect to: halui.spindle\-override.value)
 .TP
-(float in) \fIxhc\-hb04.spindle\-rps\fR Spindle speed in rps.
-(revolutions per second).  The float value is converted to a 16 bit integer
-and multiplied by 60 in order to display as RPMs,
+(float in) \fIxhc\-hb04.spindle\-rps\fR Spindle speed in RPS  (revolutions per second).
+The float value is converted to a 16 bit integer and multiplied by 60 in order to display as RPMs,
 require: 0 <= pinvalue <= 1092 (65520 RPM) (typically connect to:
 spindle.N.speed\-out\-rps\-abs)
 .TP
 (bit in) \fIxhc\-hb04.inch\-icon\fR Use inch icon (default is mm)
 
 .SH Output Pins (Status)
 .TP
-(bit out) \fIxhc\-hb04.sleeping\fR True when the driver receives a pendant
-inactive (sleeping) message.
+(bit out) \fIxhc\-hb04.sleeping\fR True when the driver receives a pendant inactive (sleeping) message.
 .TP
 (bit out) \fIxhc\-hb04.jog.enable\-off\fR True when the pendant rotary
 selector switch is in the OFF position or when the pendant is sleeping.
 .TP
-(bit out) \fIxhc\-hb04.enable\-[xyza]\fR True when the pendant rotary
-selector switch is in the [xyza] position and not sleeping.
+(bit out) \fIxhc\-hb04.enable\-[xyza]\fR
+True when the pendant rotary selector switch is in the [xyza] position and not sleeping.
```

Comment:
Why are these line break changes necessary?

Reply:
I got a sentence starting with "selector switch is in the [xyza] position and not sleeping." to translate in weblate. All those line breaks are about keeping those parts of a paragraph on a single that that make no sense if read without context. Also the highlighting is often broken into multiple llines, which is not compatible with the auto-translators.

Reply:
ok